### ISSUE TITLE
Update calendar details when modified by external app

### ISSUE THREAD

**Issue by jgriffiths** [SufficientlySecure/calendar-import-export#47]
This is instead of a refresh button as suggested by @Gitoffthelawn. Also noticed by @mzagol during offline testing of another issue.

There are two parts here, 1) reload the calendar details when the app is resumed, and 2) Register for updates from the O/S when any calendars change while running.

Note that the count may still be wrong until a workaround for issue #44 is found.

**Comment by Gitoffthelawn** [comment]
I agree. I proposed a refresh button simply because it would be less work to implement. But if you make it automatic, all the better!
